_posts/2019-07-12-moved.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+date: 2019-07-12 10:33:45
+title: "Yay! Moved to GitHub Pages"
+categories: projects
+author_name : binnur
+author_url : /author/binnur
+author_avatar: binnur
+show_avatar : true
+read_time : 3
+feature_image: feature-fish2
+show_related_posts: false
+square_related: recommend-mountain
+---
+Welcome to my new site! This is a behind the scenes look at my move off of
+WordPress. There are two significant aspects to this move:
+1. I am dusting off my blog with goal to write more
+2. After years of using WordPress, change is good, really good!!
+
+## So, why GitHub, and why now?
+> "There is virtue in work and there is virtue in rest. Use both and overlook
+neither."
+> Alan Cohen
+
+I have been enjoying my personal break and dusting off some of my old habits, which
+I miss... As I reflected on blogging, I realized I had a love/hate relationship
+with my old blog. I enjoyed verbalizing my thoughts, but it
+also felt too heavy and time consuming. Part of it was how I approached to my
+writing, more formal and deep, and part of it was the time it took to manage the
+blogging platform.
+
+So, with this move, my objective is to:
+1. Create a light-weight writing experience where I can experiment easily
+2. Simplify the publishing platform and make it fun!
+
+I have used WordPress for a long time, even hosted it myself in the early days.
+However, as an engineer at heart, I like the simplicity of git, how it handles
+versioning, and I have been looking for ways to spend more time using it. There is something deeply
+satisfying about using vim for editing, and git for check-ins.
+
+I did try to experiment with Medium for writing -- but, I didn't find it
+touching my creative side. As I type this, I realize I am using VSCode with vim
+plug-in, which part of this writing experience speaks creative?! And yes, I did install
```

Comment:
I don't understand this sentence: 'As I type this, I realize I am using VSCode with vim plug-in, which part of this writing experience speaks creative?!'
Do you mean that you are taking a creative approach to the process of writing? Saying something like 'using non-standard tools like VSCode and vim enhances my experience, and lets me offset the limitations of the tools with my creativity' or something like that?